torch/_inductor/compile_fx_ext.py

```diff
@@ -532,6 +552,7 @@ def _run_in_child(
 
 # This is a debugging/testing implementation of FxCompile which serializes the
 # input and output but still runs the FxCompile in-process.
+@final
```

Comment:
Why's this class final now?

Reply:
Just documentation that it doesn't expect anything to subclass it.